crates/torii/core/src/sql_test.rs

```diff
@@ -350,23 +360,28 @@ async fn test_get_entity_keys() {
         .await
         .unwrap();
 
-    TransactionWaiter::new(res.transaction_hash, &account.provider()).await.unwrap();
-
-    let world_reader = WorldContractReader::new(strat.world_address, account.provider());
-
-    let mut db = Sql::new(pool.clone(), world_reader.address).await.unwrap();
+    TransactionWaiter::new(spawn_res.transaction_hash, &account.provider()).await.unwrap();
 
-    let _ = bootstrap_engine(world_reader, db.clone(), account.provider()).await;
+    // Send move transaction
+    let move_res = account
+        .execute_v1(vec![Call {
+            to: actions_address,
+            selector: get_selector_from_name("move").unwrap(),
+            calldata: vec![Felt::ZERO],
+        }])
+        .send_with_cfg(&TxnConfig::init_wait())
+        .await
+        .unwrap();
 
-    let keys = db.get_entity_keys_def("dojo_examples-Moves").await.unwrap();
-    assert_eq!(keys, vec![("player".to_string(), "ContractAddress".to_string()),]);
+    TransactionWaiter::new(move_res.transaction_hash, &account.provider()).await.unwrap();
 
-    let entity_id = poseidon_hash_many(&[account.address()]);
+    let world_reader = WorldContractReader::new(strat.world_address, account.provider());
 
-    let keys = db.get_entity_keys(entity_id, "dojo_examples-Moves").await.unwrap();
-    assert_eq!(keys, vec![account.address()]);
+    let db = Sql::new(pool.clone(), world_reader.address).await.unwrap();
 
-    db.execute().await.unwrap();
+    // Expect bootstrap_engine to error out due to the existing bug
```

Comment:
```suggestion
    // Expect bootstrap_engine to not error out
```

stale comment from when i was trying to see if without the fixes it actually errors out, can be changed to this now